forge-gui/res/cardsfolder/upcoming/felonious_rage.txt

```diff
@@ -0,0 +1,8 @@
+Name:Felonious Rage
+ManaCost:R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +2 | KW$ Haste | SubAbility$ DBDelayedTrigger | SpellDescription$ Target creature you control gets +2/+0 and gains haste until end of turn. 
```

Comment:
wrong ValidTgts / Prompt

Does auto StackDescription do a good job with this? I kind of think it should, just checking